### PR TITLE
feat(autoware_launch): add use_raw_remote_control_command_input argument

### PR DESCRIPTION
**Pull request by ismetatabay**
## Description

This PR adds an option for using raw external commands for vehicle such as R/C controlled car.

autoware.universe PR: 
- https://github.com/autowarefoundation/autoware.universe/pull/4325

releated: 
- https://github.com/autowarefoundation/autoware.universe/issues/4324

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: autoware_launch/config/control/vehicle_cmd_gate/vehicle_cmd_gate.param.yaml [hunk 1]
```diff
@@ -3,7 +3,9 @@
     update_rate: 10.0
     system_emergency_heartbeat_timeout: 0.5
     use_emergency_handling: false
+    check_external_emergency_heartbeat: false
     use_start_request: false
+    enable_cmd_limit_filter: false
```
Comment (TakaHoribe) [inline autoware_launch/config/control/vehicle_cmd_gate/vehicle_cmd_gate.param.yaml:8]: Please make it true by defalut
```suggestion
    enable_cmd_limit_filter: true
```

Comment (ismetatabay) [reply autoware_launch/config/control/vehicle_cmd_gate/vehicle_cmd_gate.param.yaml]: @TakaHoribe done :+1:.

**Review by TakaHoribe — APPROVED** [review]
LGTM!

File: autoware_launch/config/control/vehicle_cmd_gate/vehicle_cmd_gate.param.yaml (hunk 1 above)
Comment (TakaHoribe) [reply autoware_launch/config/control/vehicle_cmd_gate/vehicle_cmd_gate.param.yaml]: Thanks!

**Pull request merged**